tensorboard/plugins/pr_curve/summary.py

```diff
@@ -31,7 +31,7 @@ def op(
     labels,
     predictions,
     num_thresholds=None,
-    weight=None,
+    weights=None,
```

Comment:
+1, that calls for a separate PR. https://github.com/tensorflow/tensorboard/pull/356/files